README.tr.md

```diff
@@ -0,0 +1,108 @@
+
+![This is an image](https://cdn.discordapp.com/attachments/731164914751635510/1085534409958101122/Ekran_Resmi_2023-03-15_15.05.56.png)
+# **Binabilgi.com nedir?**
+
+Binabilgi.com kişilerin olası bir depreme karşı yapıların deprem güvenliği durumlarına ilişkin ihtiyaç duyulan bilgiye ulaşmalarını sağlayan açık kaynak kodlu, harita bazlı bir aplikasyondur.  
+
+Deprem güvenliğine ilişkin farklı kaynaklardan derlenen, farklı nitelik ve ölçeklerde mekansal verileri haritalandırıp, anlaşılır ve takip edilebilir halde kullanıcıya etkin bir biçimde sunan bir mekansal veri tabanı geliştiriyoruz. 
+
+
+# **Binabilgi.com’da neyi amaçlıyoruz? Hedeflerimiz neler?**
+
+Binabilgi.com ile bilginin paylaşılması yoluyla güvenli kentleşme yolunda topluluk bilincinin yükselmesini sağlıyoruz. 
+
+Olası bir depreme karşı, yapıların deprem güvenliği durumlarına ilişkin verilerin kamu kurumları tarafından şeffaf olarak paylaşılmasını, denetimlerin yapılmasını ve güvenli, yaşamaya elverişli bir kente dair politikaların uygulamaya geçirilmesini daha güçlü bir sesle ve daha kalabalık talep etmeyi amaçlıyoruz. Bu amaçla; 
+
+* Kamu kurumlarınca üretilen farklı ölçek ve nitelikte verilerin tek bir platformda bir araya getirilmesi, 
+* Söz konusu verilerin, bir arada, anlaşılır bir şekilde haritalandırılması, 
+* İBB bina tespit analizi sonuçları ile Bakanlık tarafından yetkilendirilen Yapı Denetim Şirketlerince yapılan yapı tespit analizi sonuçlarının derlenmesi ve ulaşılabilir kılınması,
+* Her yapı bakımından bir künye oluşturulması, bu sayede sorumluluk zincirini takip edebilmeyi mümkün kılacak şekilde ilgili müteahhit şirket, mimari proje şirketi, zemin etüdü şirketi, denetim şirketi bilgilerinin derlenmesi,
+* Söz konusu verilerin güvenilirlik/ teyit edilebilirlik standartlarını sağlamasına yönelik bir doğrulama sisteminin geliştirilmesi, 
+* Yapı standart ve sertifikalarına ilişkin iyi uygulamaların görünür kılınması,
+* Yapılara projeye ve ruhsata aykırı müdehalelere ilişkin ihbar niteliğinde bilgileri ve bunlara ilişkin görsel malzemenin derlenmesi ve ilgili mercilere iletilmesini hedefliyoruz.
+
+# **Binabilgi.com nasıl işliyor?**
+
+Deprem güvenliğine ilişkin farklı ölçek ve nitelikte bilgileri tek bir künyede buluşturan binabilgi.com;
+
+## **-FAZ 1-**
+
+  1) İBB ve Kandilli Rasathanesi’nin Olası Deprem Kayıp Tahminlerinin Güncellenmesi Raporu (2020) sonuçlarına göre yapının bulunduğu mahallenin risk durumu bilgisi, 
+
+  2) İBB Kent Jeolojisi Projesi  Jeolojik Etüt Haritası (2011) uyarınca yapının bulunduğu alanın zemin bilgisi,
+
+  3)  AFAD Güncel Fay Hatları ve rilerine göre yapının bulunduğu alanın fay hatlarına uzaklığı bilgisi, 
```

Comment:
Instead, we use GEM global active faults dataset. https://github.com/GEMScienceTools/gem-global-active-faults